Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM    node:alpine
+
+RUN     mkdir /app
+WORKDIR /app
+
+ADD     . .
+RUN     npm install 
+
+VOLUME [ "/app/config.json", "/var/run/docker.sock" ]
+
+CMD     sh -c "npm run dockerkit"
```

Comment:
Please use `yarn` as package manager because mixing package managers could lead into problems. `yarn` should be installed in the `node:alpine` container so you can just switch the command!